File: PRAKTIKUM_2/5/PRAK205-2410817110010-DanielNoprianto.c
#include <stdio.h>
#include <math.h>

int main(){
//case pertama
    int a, b, c, keliling, luas;
    scanf("%d %d",&a,&b);

    c = sqrt(b*b-a*a);
    keliling = a+b+c;
    luas = 0.5*c*a;

    printf("Alas = %d cm\n",c);
    printf("Tinggi = %d cm\n",a);
    printf("Keliling = %d cm\n",keliling);
    printf("Luas = %d cm^2\n",luas);

//case kedua
    int tinggi, miring, alas, k, l;
    scanf("%d",&tinggi);
    scanf("%d",&miring);

    alas = sqrt(miring*miring-tinggi*tinggi);
    k = tinggi+miring+alas;
    l = 0.5*alas*tinggi;

    printf("Alas = %d cm\n",alas);
    printf("Tinggi = %d cm\n",tinggi);
    printf("Keliling = %d cm\n",k);
    printf("Luas = %d cm^2\n",l);
    return 0;
}

File: PRAKTIKUM_2/5/PRAK205-2410817110010-DanielNoprianto.py
import math
#case pertama
tinggi, sisi_miring=map(int, input().split())
alas = math.sqrt(sisi_miring**2-tinggi**2)
keliling = alas+tinggi+sisi_miring
luas = 0.5*alas*tinggi

print(f"Alas= {alas:.0f} cm")
print(f"Tinggi= {tinggi:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas  = {luas:.0f} cm^2")

#case 2
a = float(input("\n"))
b = float(input(""))
c = math.sqrt(b**2-a**2)
k = a+b+c
l = 0.5*c*a

print(f"Alas= {c:.0f} cm")
print(f"Tinggi= {a:.0f} cm")
print(f"Keliling = {k:.0f} cm")
print(f"Luas  = {l:.0f} cm^2")